test/util.ts

```diff
@@ -87,6 +87,7 @@ export function expectEachVersionExceptJit<T>(
         [tstl.LuaTarget.Lua53]: expectation,
         [tstl.LuaTarget.Lua54]: expectation,
         [tstl.LuaTarget.LuaJIT]: false, // Exclude JIT
+        [tstl.LuaTarget.Luau]: expectation,
```

Comment:
The reasoning for this function is that we are running WASM-versions of the lua runtimes (5.0-5.4) during our test runs, but we do not have JIT running. So this function is mostly for asserting things about the execution results in the Lua runtime, hence I think just putting Luau to false here is probably more appropriate